eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -1890,6 +1889,20 @@ public function loadRelations(APIVersionInfo $versionInfo)
         return $relations;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function countReverseRelations(ContentInfo $contentInfo): int
+    {
+        if (!$this->repository->canUser('content', 'reverserelatedlist', $contentInfo)) {
+            return 0;
```

Comment:
What is the use case for not throwing Unauthorized exception here? It's not a filtered list.

Reply:
I think we discuss this in the previous PR about the content draft and we decided that a user should always receive count. Because he will get it when he will fetch the paginated list. In the list total count is present.